style.css

```diff
@@ -128,9 +124,10 @@ pre {
   border-color: #ada;
   background: #efe;
   padding: 0 40px;
-  font-size: 85.714%;
-  line-height: 1.334;
-  font-family: "Courier", monospace;
+  font-size: 76.191%;
+  line-height: 1.5;
+  font-family: "Fira Code", "Courier", monospace;
+  font-variant-ligatures: none;
```

Comment:
I appreciate ligatures for static text but I don't enjoy magical things happening as I type.

Reply:
I use it in my editor and find it quite nice. But I can see your point.

Reply:
We can easily remove this line in the future if you change my mind. :)